planner/core/hints.go

```diff
@@ -245,21 +245,23 @@ func extractTableAsName(p PhysicalPlan) (*model.CIStr, *model.CIStr) {
 
 func getJoinHints(sctx sessionctx.Context, joinType string, parentOffset int, nodeType nodeType, children ...PhysicalPlan) (res []*ast.TableOptimizerHint) {
```

Comment:
Do the aggregation hints have the same problem?

Reply:
I think no, because in agg hints, there is no place for tables.